content/css/concepts/flexbox/terms/flex-basis/flex-basis.md

```diff
@@ -45,3 +45,8 @@ An image that has a starting width of 200 pixels:
   flex-basis: 200px;
 }
 ```
+<!-- Beginning of Issue -->
+<!-- Add a output image for the example already given in the entry. -->
+See below use of `#img_one` alongside others with default `flex` properties:
+
+![Image demonstrating example 1 code alongside other flex elements with default properties.](../../../../../../media/css-flex-basis-example-output.png)
```

Comment:
```suggestion
![Image demonstrates the use of 200px `flex-basis` on `#img_one` alongside other images in the flex with default properties.](../../../../../../media/css-flex-basis-example-output.png)
```
Make the suggested change to the `alt-text` of the image.

Also, change the image link to the following format:
"https://raw.githubusercontent.com/Codecademy/docs/main/media/" followed by the image name.